pkg/kv/bulk/buffering_adder.go

```diff
@@ -94,6 +94,8 @@ func MakeBulkAdder(
 		// splitting _before_ hitting max reduces chance of auto-splitting after the
 		// range is full and is more expensive to split/move.
 		opts.SplitAndScatterAfter = func() int64 { return 48 << 20 }
+	} else if opts.SplitAndScatterAfter() == -1 {
+		opts.SplitAndScatterAfter = nil
```

Comment:
nit:
Could we add a comment on `SplitAndScatterAfter` that returning -1 indicates that it will not split at all, but not specifying it will fallback to some default value?

Maybe introducing a `const ForceNoSplitAndScatter = -1` in this package would be beneficial for readability?

Reply:
Good point, done.

Reply:
Let's also use the const in the check above?